web/components/logos/NavBarLogo.tsx

```diff
@@ -0,0 +1,50 @@
+import Link from 'next/link';
+import Image from 'next/image';
+import navbar_logo from "../../public/navbar_logo.png";
+import navbar_logo_bg from "../../public/navbar_logo_bg.png"
+
+interface LogoProps {
+  isAdmin: boolean;
+  caregiverName?: string;
+}
+
+const Logo: React.FC<LogoProps> = ({ isAdmin, caregiverName }) => {
+  const homeLink = isAdmin ? '/admin' : '/caregiver';
+  
+  return (
+    <div>
+        <Link href={homeLink}>
+        <button>
+            <span className="left-0 right-0 w-[318px] h-[81px]">
+                <Image className="relative" src={navbar_logo_bg} />
+            </span>
+            <span className="absolute top-[26px] left-[33px]">
+                <Image className="static px-10" src={navbar_logo} />
+            </span>
+            {
+                isAdmin ? 
+                    <span className="absolute top-[29px] left-[82px]">
```

Comment:
This shouldn't have absolute position because it looks different on different screens, so make sure to set these positions relatively!